chaoscenter/cypress/e2e/negative_tests.cy.js

```diff
@@ -0,0 +1,469 @@
+import { add_hub } from "../fixtures/chaoshub";
```

Comment:
It would be better to have negative tests in same file where we have positive tests.
One file per feature would be better.